### DISCUSSION TITLE
Journal vs. Diary

### DISCUSSION THREAD

**Discussion by Dricc123** [jan-mate/Journalize#3] [Q&A] [answered]
Hello, I find it interesting that you make a distinction between a journal and a diary and that you suggest diary apps for those looking for a diary. What's the difference?

**Comment by jan-mate** [comment, marked as answer]
Hi there! That’s a great question. Although many people use the terms diary and journal interchangeably, I personally see a distinction:

- A diary tends to be more structured, where people often write at the end of the day to document events, emotions, and experiences. It usually follows a daily format, with entries for specific dates.

- A journal, on the other hand, is more flexible. You can write whenever inspiration strikes, capturing thoughts you want to reflect on or revisit later. It’s less bound to a specific schedule and more about ongoing contemplation.

The reason I recommend other apps as diaries is because they have built in calendars, which makes them more structured.
Whilst I prefer Journalize for journalling because it allows me to quickly write down something, tag it, and later search and revisit the thought.
But you can definitely use the mentioned diary apps for journalling, but in my opinion they're not as quick to use, which is why I made my own app. On the other hand, Journalize can be used as a diary, but you can't view your entries in chronological order.

**Reply by Dricc123** [reply]
Very interesting. Now I'm thinking whether or not I need a calendar for my use case. Maybe not! As long as there is a date timestamp, which Journalize has.
I can't view my entries in Chronologic order, you say? aren't they sorted chronologically in the entries list view?

**Reply by jan-mate** [reply]
They're sorted by Last Modified timestamp, so if you reopen previous entries it messes up the chronilogical order. 
I can see the how using the app as a diary will be easier if there's chronological sorting. I may add that as a toggle, should be easy to implement.

**Reply by jan-mate** [reply]
I added the option to search by creation date in v1.1.0 . This should make the app work better as a diary.
